include/circt/Dialect/SV/SVExpressions.td

```diff
@@ -91,3 +91,22 @@ def ConstantZOp : SVOp<"constantZ", [NoSideEffect, HasCustomSSAName]> {
   let results = (outs HWIntegerType:$result);
   let assemblyFormat = " attr-dict `:` type($result)";
 }
+
+def LocalparamOp : SVOp<"localparam",
```

Comment:
Should this be "LocalParamOp"?  I know the keyword is all lower case, but we typically capitalize words